Makefile

```diff
@@ -193,6 +193,31 @@ GCOV2LCOV := $(GO) run github.com/jandelgado/gcov2lcov
 
 GO_TEST_MIN_COVERAGE := 50
 
+##
+## protobuf
+##
+
+PROTOC_VERSION := 28.0
```

Comment:
@ricardo6142dh please check https://github.com/fornellas/resonance/pull/117.

We need to figure out a way to keep this updated. My suggestion here, is to:

- Save this version on a `.protoc_version` file, and populate this variable from the file contents.
- Add a `update-protoc` target, that updates the version at `.protoc_version`, and tie to teh new target from #117 .

To get the latest version, somethnig like this should do:

```
curl -v https://github.com/protocolbuffers/protobuf/releases/latest/ 2>&1 | grep -Ei '^< location: ' | tr / \\n | tail -n 1
```
but it of course needs some polishing to take care of errors (eg: what if there's a 404, it comes up empty, etc etc?).

Reply:
It will be solved in a separeted PR